data/lm/generate_package.py

```diff
@@ -62,7 +62,8 @@ def create_bundle(
     scorer.fill_dictionary(list(words))
     shutil.copy(lm_path, package_path)
     # append, not overwrite
-    if scorer.save_dictionary(package_path, True):
+    save_dic = scorer.save_dictionary(package_path, True)
+    if save_dic is None or save_dic:
```

Comment:
Why should we support multiple versions of the Scorer code here? We've never done this, instead instructing users to install matching versions of packages/code/models.

Reply:
Maybe we should make generate_package.py run some version checks like we did in train.py.

Reply:
> Why should we support multiple versions of the Scorer code here? We've never done this, instead instructing users to install matching versions of packages/code/models.

We have a misleading error message here. Do you feel confident releasing a 0.7.4 with all the changes that happened? That's the only correct solution, I agree.

Reply:
We only have a misleading error message if the user is using mixed versions of generate_package.py and ds-ctcdecode package, no?

Reply:
> We only have a misleading error message if the user is using mixed versions of generate_package.py and ds-ctcdecode package, no?

Yes, which is the case if people do follow docs on current master, so we'll get people complaining and adding noise here :/